<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Lens name</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
     <property name="background">
      <color>
       <red>180</red>
       <green>27</green>
       <blue>45</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delta per step</string>
     </property>
     <property name="background">
      <color>
       <red>238</red>
       <green>238</green>
       <blue>236</blue>
      </color>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_freq_down">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_freq_up">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_handle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 255, 88);</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_handle">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 65, 116);</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QWidget" name="tree_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>181</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="handles_table">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Handle name</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
     <property name="background">
      <color>
       <red>180</red>
       <green>27</green>
       <blue>45</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="handle_info">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Handle info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
